src/travertino/node.py

```diff
@@ -1,9 +1,7 @@
 class Node:
-    def __init__(self, style, applicator=None, children=None):
+    def __init__(self, style=None, applicator=None, children=None):
```

Comment:
Allowing style to be None is an interesting choice - what's the reasoning here?

From a high level perspective, I'd argue that we should be able to guarantee that a node has a style - otherwise we don't have any basis to make *any* layout-based calculations (cf - in this code, until you set the style, you don't have any layout or intrinsic sizing). 

The *applicator* will (usually) need to be deferred, because we can't apply until we know the node is fully configured, but the even if there's a completely *default* style, there will still be a style.

Is there an edge case here that I'm missing?

Reply:
...I thought I'd left inline comments of my own and replied to some of these, but apparently they were all part of a review I'd started by accident. Ah, Github.

No, I'm the one missing the forest for the trees here. Unaware of the need for a style upfront, I was just assuming that, now that style and applicator both trigger application when whichever is assigned, it didn't matter what, if anything, was first.

Reply:
Also, in which case: should we prohibit setting style to `None`?

Reply:
I don't think we need to be proactive about it - things will explode nicely enough on their own when you try to access attributes on the style.